remote/src/main/scala/org/apache/pekko/remote/transport/PekkoProtocolTransport.scala

```diff
@@ -136,7 +136,7 @@ private[remote] class PekkoProtocolTransport(
   }
 
   override val maximumOverhead: Int = PekkoProtocolTransport.PekkoOverhead
-  protected def managerName = s"akkaprotocolmanager.${wrappedTransport.schemeIdentifier}${UniqueId.getAndIncrement}"
+  protected def managerName = s"pekkoprotocolmanager.${wrappedTransport.schemeIdentifier}${UniqueId.getAndIncrement}"
```

Comment:
can this be `pekko-protocol-manager` ?

Reply:
I want to maintain the style of the legacy name which is `akkaprotocolmanager`. I don't think adding `-` chars makes them a great deal more readable.